python/py-scgi/Portfile

```diff
@@ -1,23 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-scgi
-version             1.14
+version             1.15
 maintainers         nomaintainer
 platforms           darwin
 
 description         A Python package for implementing SCGI servers.
 long_description    ${description}
 license             CNRI MIT
-homepage            https://pypi.python.org/pypi/scgi/
+homepage            https://python.ca/scgi/
 
-livecheck.regex     {>scgi (.+) :}
-livecheck.type      regex
-livecheck.url       ${homepage}
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
 
-distname            scgi-${version}
-master_sites        https://python.ca/scgi/releases/
+checksums           rmd160  515d5d93a7195395f6a51f5075e2e9438c91b6eb \
+                    sha256  d0a85722bf0417aad07a0623e47e41ad7de65868cf168fac2a84446b5303a3ed \
+                    size    28166
 
-checksums           rmd160 8214b939d52c6ba7e4392c70efa829d05941fc25
+python.versions     27
 
-python.versions     26
+if {$subport ne $name} {
```

Comment:
Usually we use curly brackets for variables - e.g. `if {${subport} ne ${name}}`

By the way, any reason to not check against PyPI?

Reply:
see above for livecheck. I didn't change that line I think, but can add the curly brackets if you prefer.

Reply:
okay, curly brackets added